adapter/internal/oasparser/model/async_api.go

```diff
@@ -178,10 +178,9 @@ func populatePoliciesFromVendorExtensions(operation *Operation, vendorExtensions
 	policyParameters[constants.RewritePathResourcePath] = newResourcePath
 	policyParameters[constants.IncludeQueryParams] = true
 	policy := Policy{
-		PolicyName: constants.RewritePathResourcePath,
-		Action:     constants.RewritePathTemplate,
-		Order:      1,
-		Parameters: policyParameters,
+		Action:           constants.RewritePathAction,
```

Comment:
Are we not reading the policy name from the vendor extensions?

Reply:
Nope, this is not read from vendor extensions, Action is read from the policy definition go template file. But in here we have hard coded the policy "RewritePath" @VirajSalaka I am correct, right?